packages/debian/rules

```diff
@@ -2,13 +2,16 @@
 
 include /usr/share/dpkg/pkg-info.mk
 
-INIT_SYSTEM ?= systemd
-export PYBUILD_INSTALL_ARGS=--init-system=$(INIT_SYSTEM)
+export PYTHONDONTWRITEBYTECODE=1
 
 %:
-	dh $@ --with python3 --buildsystem pybuild
+	dh $@ --buildsystem meson
+
+override_dh_auto_configure:
+	dh_auto_configure -- -Dinit_system=systemd -Dlibexecdir=lib
```

Comment:
We don't have to, but we could use the templates option here too.

Reply:
done for debian, redhat  and suse templates.